exercises/markdown/example.py

```diff
@@ -22,9 +22,9 @@ def wrap(line, tag):
 
 def check_headers(line):
     pattern = '# (.*)'
-    for i in range(6):
+    for index in range(6):
```

Comment:
Out of curiosity: does `flake8` complain about this one, or was that your personal style preference?

Reply:
Apologies - for whatever reason, this comment did not get emailed to me...so I am only seeing it now.  That change is a personal preference.  I can always refrain if it is bugging you/not good practice in this situation.

Reply:
Doesn't bother me; I just see `for i in range(n)` so frequently that I generally give that one a pass on variable naming rule.